CareKit/CareKit/iOS/Task/Controllers/OCKSliderTaskController.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  OCKSliderTaskController.swift
+//  
+//
+//  Created by Dylan Li on 5/26/20.
+//  Copyright © 2020 NetReconLab. All rights reserved.
+//
+
+#if !os(watchOS)
+
+import CareKitStore
+import Combine
+import Foundation
+import SwiftUI
+
+open class OCKSliderTaskController: OCKTaskController {
+    
+    /// Data used to create a `CareKitUI.SliderTaskTaskView`.
+    @Published public private(set) var viewModel: SliderTaskViewModel? {
+        willSet { objectWillChange.send() }
+    }
+    
+    /// Data used to create a `CareKitUI.SliderTaskTaskView`.
+    @Published public private(set) var value: CGFloat = 0 {
+        willSet { objectWillChange.send() }
```

Comment:
Maybe the DispatchQueue line in the comment below could go in the didSet of this but I was not sure how to implement this